Adyen/UI/List/CoreListDataSource.swift

```diff
@@ -39,7 +39,7 @@ internal protocol ListViewControllerDataSource: UITableViewDataSource {
     func stopLoading(_ tableView: UITableView)
 }
 
-internal final class CoreDataSource: NSObject, ListViewControllerDataSource {
+internal final class CoreListDataSource: NSObject, ListViewControllerDataSource {
```

Comment:
Do we really need `Core` in it? Why not just `ListDataSource`? Or `DynamicListDataSource` ?

Reply:
because `DiffableListDataSource` uses `CoreListDataSource` behind the scenes, so `CoreListDataSource` is the "Core" one :-D.

and I like `DiffableListDataSource` more than `DynamicListDataSource`, because both data sources are dynamic, one of them subclasses `UITableViewDiffableDataSource`, hence the `Diffable` in the name.

Reply:
oh, make sense :)